test/com/facebook/buck/android/testdata/android_instrumentation_test_integration_test/apps/myapp_test/BUCK

```diff
@@ -0,0 +1,14 @@
+android_instrumentation_apk(
+  name = 'app_test',
+  manifest = 'AndroidManifest.xml',
+  apk = '//apps/myapp:app-exo',
+  deps = [
+    '//test/com/example/activity:activity_test',
+  ],
+)
+
+android_instrumentation_test(
+  name = 'test',
+  apk = '//apps/myapp:app-exo',
```

Comment:
hmm I'm not sure I follow.  You mean just removing the `apk` target here and finding the main apk from the android_instrumentation_apk() target if necessary?

If that's what you mean it shouldn't be too hard, we could just store the apkUnderTest in the AndroidInstrumentationApk class, and then get it if we have an `android_instrumentation_apk()` as `instrumentation_apk`.

Or did you mean something else?